README.md

```diff
@@ -136,6 +135,22 @@ value should be less than ~5000, otherwise fuzzer can miss new interesting input
 due to hash collisions. And finally ```uptime``` is uptime of the process. This same
 information is also served via http (see the ```-http``` flag).
 
+## Modules support
+
+go-fuzz has preliminary support for fuzzing [Go Modules](github.com/golang/go/wiki/Modules). 
+go-fuzz respects the standard `GO111MODULE` environment variable, which can be set to `on`, `off`, or `auto`. 
+
+You must have a `require` directive for `github.com/dvyukov/go-fuzz` in your go.mod file. Two approaches are:
+
+1. Prior to invoking go-fuzz-build, add a `require` for `github.com/dvyukov/go-fuzz` in your go.mod. A common approach for tracking tool dependencies like this is [via a tools.go file](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module). Alternatively, you could for example manually add `require github.com/dvyukov/go-fuzz latest` to your go.mod.
+
+2. If you do not have a `require` for `github.com/dvyukov/go-fuzz` in your go.mod, go-fuzz-build will add one automatically. If desired, you can revert your go.mod file when go-fuzz-build is done.
+
+Vendoring with modules is not yet supported. A `vendor` directory will be ignored, and go-fuzz will report an error if `GOFLAGS=-mod=vendor` is set.
+
+Note that while modules are used to prepare the build, the final instrumented build is still done in GOPATH mode.
```

Comment:
Is it important to tell people this in the README? Seems like an implementation detail.

Reply:
Two reasons I added this: 

1. There might be some small number of projects out there that check `GO111MODULE` directly when executing. For example, at least for a time, I think go-buffalo did this when doing some type of code generation, though I don't know if someone would ever fuzz that step, and I don't use go-buffalo myself so I don't know exactly what it was doing by checking `GO111MODULE` .  This is therefore a warning of a limitation in the current implementation, though my guess is it would be rare for it to be an actual issue.

2. This is the main reason this is "preliminary" modules support. At some point, GOPATH is almost certainly going away, and when that happens, go-fuzz will break as currently implemented on master without or without this PR. There is some smallish chance this might help some future gopher who might not yet be a contributor to go-fuzz, or maybe it helps some future problem reporter when things break due to some future change in modules.

That said, happy to remove.